bubble-wrap-style.yaml

```diff
@@ -5392,28 +5411,125 @@ layers:
+                            stroke: { width: 0px }
+
+            width_1char:
+                filter: function() { return (feature.bus_shield_text.length === 1); }
+                draw:
+                    mapzen_icon_library:
+                        size: [[6,[16px,16px]],[13,[18px,18px]],[16,[21px,21px]]]
+            width_2char:
+                filter: function() { return (feature.bus_shield_text.length === 2); }
+                draw:
+                    mapzen_icon_library:
+                        size: [[6,[21px,16px]],[13,[24px,18px]],[16,[28px,21px]]]
+            width_3char:
+                filter: function() { return (feature.bus_shield_text.length === 3); }
+                draw:
+                    mapzen_icon_library:
+                        size: [[6,[27px,16px]],[13,[31px,18px]],[16,[36px,21px]]]
+            width_4char:
+                filter: function() { return (feature.bus_shield_text.length === 4); }
+                draw:
+                    mapzen_icon_library:
+                        size: [[6,[32px,16px]],[13,[36px,18px]],[16,[41px,21px]]]
+                # London sucks
+                early:
+                    filter: { $zoom: { max: 14 } }
+                    draw:
+                        mapzen_icon_library:
+                            visible: false
+            width_5char:
+                filter: function() { return (feature.bus_shield_text.length === 5); }
+                draw:
+                    mapzen_icon_library:
+                        size: [[6,[37px,16px]],[13,[41px,18px]],[16,[48px,21px]]]
+                # London sucks
+                early:
+                    filter: { $zoom: { max: 14 } }
+                    draw:
+                        mapzen_icon_library:
+                            visible: false
+            # this is kinda a hack
+            width_longggggg:
+                filter: function() { return (feature.bus_shield_text.length > 5); }
+                draw:
+                    mapzen_icon_library:
+                        visible: false
+ 
+        z19-show-operator-name:
+                filter:
+                    all:
+                        - function() { return global.sdk_transit_overlay; }
+                        - $zoom: { min: 19 }
+                draw:
+                    text:
+                        text_source: function() { return (feature.bus_network); }
+                        priority: 10.99
+                        font:
+                            fill: [0,0,0.5,0.8]
+                            style: italic
+                            size: 14px
+                            stroke: { color: white, width: 3px }
 
     transit-overlay-station-labels:
         data: { source: mapzen, layer: pois }
         filter:
             kind: [station, tram_stop, bus_station, subway_entrance, halt, stop, platform, bus_stop, stop_position ]
             any:
                 - area: false
```

Comment:
What's going on with both `area: false` and `area: true`?

Reply:
That's odd, probably a holdout from around v1.0 tiles? 

In any event, removing it doesn't break anything and simplifies the style: eafad2608ad9a4ba8510daabd71556d99c504928.